README.md

```diff
@@ -1,4 +1,34 @@
 # 🔥_Fireselect_
+Vue component to `<select>` box.  
+It's a lot like Selectize, Chosen, Select2 built only with Vuejs ;p
```

Comment:
Suggestion:
It is similar to Selectize, Chosen, Select2, etc. however it was built using Vue.js only ;)